environment.go

```diff
@@ -23,6 +25,8 @@ func resetEnv() { globalEnv = &envState{name: "global", conf: &Configuration{}}
 type Environment interface {
 	Configure(*Configuration) error
 
+	AddAuth(gimlet.UserManager, time.Duration)
```

Comment:
expire after should be configurable in the DB config struct.

Reply:
(also in general, given a config struct or a subset of it, the production of a user manager should be lightweight and not the kind of thing that we need to cache like this.